www/styles/docs.scss

```diff
@@ -226,20 +242,49 @@
     display: grid;
     grid-template-columns: 1fr;
     align-items: flex-start;
-    gap: var(--site-gap);
+    gap: calc(var(--site-gap) / 2);
     @media (min-width: breakpoint(tablet)) {
       grid-template-columns: [side-start] var(--nav-width) [side-end content-start] 1fr [content-end];
     }
+    @media (min-width: breakpoint(wrapper)) {
+      grid-template-columns: [side-start] var(--nav-width) [side-end content-start] 1fr [content-end toc-start] var(--toc-width) [toc-end];
```

Comment:
Still can't get over how readable these column names are. I'm new to named columns, and clearly need to use them more 👏